### ISSUE TITLE
Bob cannot bundle with certain characters in the project title

### ISSUE THREAD

**Issue by JCash** [defold/defold#8355]
As reported [here](https://forum.defold.com/t/error-failed-compiling-android-resources/68060/7), the bundling step may fail if the title contains certain characters.

From forum post:

 If I set project title to “Kỳ Hoàng”, it failed to compile Android bundle, but if I set it to “Ky Hoang”, it compiled successfully.

These lines are in console when it failed:

```
2024-01-02 12:30:20 INFO exec: C:\Users\chung\AppData\Local\Temp\3013256825095741087\aapt2.exe compile -o C:\Users\chung\AppData\Local\Temp\compiled_resources9302001017514913233\com.chungxa.kyhoang --dir D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\res\com.chungxa.kyhoang
```

**Comment by baochungit** [comment]
Could someone take this? Thanks!

**Comment by britzl** [comment]
> Could someone take this? Thanks!

It has not been planned yet. I took a quick look and was unable reproduce on macOS. Which OS are you using? Also, is this when running from the editor or from command line using bob.jar?

**Comment by baochungit** [comment]
Thanks for taking your time! I'm using Windows 11 and that was running from editor.

**Comment by baochungit** [comment]
I added some images to describle my case
<img width="960" alt="Screenshot 2024-01-30 101107" src="https://github.com/defold/defold/assets/228385/c24beae1-25d6-4326-be24-6f7caac2f771">
<img width="960" alt="Screenshot 2024-01-30 101138" src="https://github.com/defold/defold/assets/228385/93fb9f2b-1e98-4179-b6f5-62c6f53727c4">
<img width="960" alt="Screenshot 2024-01-30 101340" src="https://github.com/defold/defold/assets/228385/95c309b0-d6e2-41fb-b77d-228175326d46">

It seems the error message now is different from the one I catched on 1.6.2 but it still failed to build with the project title having unicode characters, for example `Kỳ Hoàng`.

---
2024-01-30 10:13:05 INFO    Copying resources to D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\aab\base\res 

2024-01-30 10:13:05 INFO    Copying engine to D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\aab\base\lib\armeabi-v7a\libKHong.so 

2024-01-30 10:13:05 INFO    Zipping D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\aab\base to D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\aab\base.zip 

2024-01-30 10:13:06 INFO    Creating Android Application Bundle 

2024-01-30 10:13:06 INFO    exec: D:\Defold\packages\jdk-17.0.5+8\bin\java -jar C:\Users\chung\AppData\Local\Temp\14482672558673546865\bundletool-all.jar build-bundle --modules D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\aab\base.zip --output D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\Kỳ Hoàng.aab --config D:\Defold\Projects\KyHoang\build\armv7-android\Kỳ Hoàng\aab\BundleConfig.json 

ERROR: unspecified:-1: 'com.dynamo.bob.CompileExceptionError: Failed creating Android Application Bundle'

**Comment by baochungit** [comment]
Will I wait for build server or Defold Editor updated to have this fixed?

**Comment by britzl** [comment]
This is a change in bob.jar. It should be available now.

**Comment by baochungit** [comment]
Sorry but I'm quite confused since I'm still seeing this error when building Android apk on Defold 1.7.0.
What should I do to have this change working?

**Comment by britzl** [comment]
This was merged last week so it is not included in bob.jar for 1.7.0. It is also not included in 1.8.0 since the beta is too old. It will be included in 1.8.1, but you can try it now by downloading the bob.jar alpha for 1.8.1:

https://github.com/defold/defold/releases/tag/1.8.1-alpha
